stacks/serverless/cms-audio-lambda.yml

```diff
@@ -0,0 +1,184 @@
+# stacks/serverless/cms-audio-lambda.yml
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Audio processing for cms.prx.org
+Parameters:
+  AlarmWarnActions:
+    Type: CommaDelimitedList
```

Comment:
This may or may not work.  But simplifies this template to put the ops-warn vs ops-error logic in the root template.